### DISCUSSION TITLE
Hebrew documents or websites

### DISCUSSION THREAD

**Discussion by freshuk** [Azure-Samples/chat-with-your-data-solution-accelerator#1332] [General]
Hi, I can't seem to upload any document in Hebrew. Nor pdf or URLs with Hebrew contents. The error says something about Latin encoding?
Thanks

**Comment by Roopan-Microsoft** [comment]
Hey @freshuk - This issue is resolved. Please retry and let us know your feedback.

Please feel free to open any issue if you are facing, we are happy to help!
